controller/embedded/api-schema/entities/terminator.put.json

```diff
@@ -20,6 +20,12 @@
     "address": {
       "type": "string"
     },
+    "cost": {
+      "type": [
+        "integer",
```

Comment:
Are there valid values for this?

IIRC JSON parsing will provide an `float64` unless the `Number` type is used, we force it to `int` and then `uint16`. If I am reading this right, I can put values in here that will default to 0.

Might want to at least include `"minimum": 0` and `"maximum": 65535`

Reply:
Fixed